samples/web-nodejs-with-db-sample/devfile.yaml

```diff
@@ -6,6 +6,7 @@ projects:
     source:
       type: git
       location: 'https://github.com/ijason/NodeJS-Sample-App.git'
+      commitId: 187d468
```

Comment:
Maybe it makes sense to add comment what this commitId means? Like the last commit that uses express-3....

Reply:
done